doc/source/ray-air/examples/pytorch_resnet_finetune.ipynb

```diff
@@ -0,0 +1,917 @@
+   "source": [
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "checkpoint_folder = result.checkpoint.uri[7:]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ray.train.batch_predictor import BatchPredictor\n",
+    "from ray.train.torch import TorchCheckpoint, TorchPredictor\n",
+    "\n",
+    "ckpt = TorchCheckpoint.from_directory(checkpoint_folder)\n",
+    "predictor = BatchPredictor.from_checkpoint(\n",
+    "    ckpt, TorchPredictor, model=initialize_model()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-02-14 14:41:29,186\tINFO batch_predictor.py:184 -- `num_gpus_per_worker` is set for `BatchPreditor`.Automatically enabling GPU prediction for this predictor. To disable set `use_gpu` to `False` in `BatchPredictor.predict`.\n",
+      "2023-02-14 14:41:29,470\tWARNING compute.py:549 -- Requested batch size 4096 results in batches of 2466299904 bytes for map_batches tasks, which is larger than the configured target max block size 536870912. This may result in out-of-memory errors for certain workloads, and you may want to decrease your batch size or increase the configured target max block size, e.g.: from ray.data.context import DatasetContext; DatasetContext.get_current().target_max_block_size = 4_000_000_000\n",
+      "2023-02-14 14:41:29,470\tWARNING compute.py:520 -- `batch_size` is set to 4096, which reduces parallelism from 153 to 1. If the performance is worse than expected, this may indicate that the batch size is too large or the input block size is too small. To reduce batch size, consider decreasing `batch_size` or use the default in `map_batches`. To increase input block size, consider decreasing `parallelism` in read.\n",
+      "Map_Batches:   0%|          | 0/1 [00:00<?, ?it/s]2023-02-14 14:41:29,588\tWARNING worker.py:1851 -- Warning: The actor BlockWorker is very large (42 MiB). Check that its definition is not implicitly capturing a large array or other object in scope. Tip: use ray.put() to put large objects in the Ray object store.\n",
+      "Map Progress (1 actors 1 pending): 100%|██████████| 1/1 [00:08<00:00,  8.41s/it]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "predictions: extension<arrow.py_extension_type<ArrowTensorType>>\n",
+      "label: int64\n",
+      "[{'predictions': array([ 0.6944345, -2.0196059], dtype=float32), 'label': 0}]\n"
+     ]
+    }
+   ],
+   "source": [
+    "prediction_ds = predictor.predict(\n",
+    "    ray_datasets[\"val\"],\n",
+    "    feature_columns=[\"image\"],\n",
+    "    keep_columns=[\"label\"],\n",
+    "    num_gpus_per_worker=1,\n",
+    ")\n",
+    "print(prediction_ds.schema())\n",
+    "print(prediction_ds.take(1))"
```

Comment:
Show these things in separate cells. One cell for each + a text block before each talking about what this is showing.